builder/ebs/builder_acc_test.go

```diff
@@ -549,6 +549,10 @@ func TestAccBuilder_EbsSessionManagerInterface(t *testing.T) {
 				return fmt.Errorf("SSH key was uploaded, but shouldn't have been")
 			}
 
+			if strings.Contains(string(logs), "Bad exit status: -1") {
```

Comment:
Just thought about the exit code here, I wonder if it'll always be `-1` and if it may not change in the future for some reason. I think we can be slightly more resilient by only checking for `Bad exit status` in the output here.